src/Export/Logger/ExportExceptionLogger.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\FinSearch\Export\Logger;
+
+use BadMethodCallException;
+use FINDOLOGIC\Export\Exceptions\EmptyValueNotAllowedException;
+use FINDOLOGIC\FinSearch\Exceptions\Export\Product\AccessEmptyPropertyException;
+use FINDOLOGIC\FinSearch\Exceptions\Export\Product\ProductHasNoAttributesException;
+use FINDOLOGIC\FinSearch\Exceptions\Export\Product\ProductHasNoCategoriesException;
+use FINDOLOGIC\FinSearch\Exceptions\Export\Product\ProductHasNoNameException;
+use FINDOLOGIC\FinSearch\Exceptions\Export\Product\ProductHasNoPricesException;
+use FINDOLOGIC\FinSearch\Exceptions\Export\Product\ProductInvalidException;
+use Psr\Log\LoggerInterface;
+use Shopware\Core\Content\Product\ProductEntity;
+use Throwable;
+
+class ExportExceptionLogger
```

Comment:
This little refactoring was done to have easier exception handling and logging. It wouldn't deserve its own story as the change is a bit minor but makes life easier for debugging, as the exception itself is now part of the logged thing as well. This was only done before for `InvalidProductException`, but now is done for all exceptions that may be thrown.